merkle-tree/src/db/service/leaf.service.js

```diff
@@ -41,12 +42,13 @@ export default class LeafService {
 
   /**
   Insert many leaves into the merkle tree
-  @param {array} data - an array of leaf objects ('documents')
+  @param {number} treeHeight
+  @param {array} leaves - an array of leaf objects ('documents')
   */
-  async insertLeaves(data) {
+  async insertLeaves(treeHeight, leaves) {
     console.log('\nsrc/db/service/leaf.service insertLeaves()');
-    //   console.log('data before mapping:', data);
-    const mappedData = data.map(item => leafMapper(item));
+    console.log('data before mapping:', leaves);
```

Comment:
This is all really good - thank you for cleaning up a lot of my bodgy treeHeight code!
A query on this bit - I usually comment out these logs because they're quite long, should they be left uncommented?